sv.c

```diff
@@ -17487,6 +17492,47 @@ S_find_uninit_var(pTHX_ const OP *const obase, const SV *const uninit_sv,
         goto do_op;
     }
 
+    case OP_LENGTH:
+        o = cUNOPx(obase)->op_first;
+        sv = find_uninit_var(o, uninit_sv, match, desc_p);
+        if (sv) {
+            Perl_sv_insert_flags(aTHX_ sv, 0, 0, STR_WITH_LEN("length("), 0);
```

Comment:
Because the macro form takes 6 arguments, but I'm only passing 5. `STR_WITH_LEN("...")` expands to two comma-separated items, but that happens too late.

Reply:
Ugh, ohgod yes that particular issue. I wonder if we should make some conveneince wrappers for this. Perhaps:

```
sv_prepend_pvn(sv, pv, len, flags)
sv_prepend_pvs(sv, "str", flags)
```

A separate issue though; this use here is fine given that.